app/views/split_checkout/_your_details.html.haml

```diff
@@ -1,5 +1,6 @@
 %ng-form{"ng-controller" => "DetailsCtrl", name: "details"}
   %div.checkout-substep
+    %input{type: :hidden, id: "advance_to_state", "name": "advance_to_state", "value": "delivery"}
```

Comment:
@jibees this is still a WIP but I thought I'd push to get your thoughts. We'll either need to have the form pass a parameter to the server like this, or have the form on Step 1 point to a different endpoint entirely. I think this first approach (passing a param) will work best, but open to other ideas. Note that this parameter doesn't actually get sent right now, I didn't want to dive into the Angular code :)

Reply:
Don't know which approach is simpler or even logical, I trust you. 
Will go into the AngularJS code (😢) to send the the parameter ;)

Reply:
I've committed: 7dd4889
This is a WIP, but it seems that, after a client validation, the form is then submitted into the `SplitCheckoutController`

Reply:
> %input{type: :hidden, id: "advance_to_state"

Just thinking out loud; does the client-side really need to tell the server-side where it should be going next or what it should be doing? Or does that responsibility lie with the server, and the client can be less "smart" and just post it's data and let the server direct/tell the client where to go or what to do next? Know what I mean?

Reply:
There will be some interesting cases here, for example if the user has already filled out all the steps but then _goes back_ to change the address. In that case when the form is posted again on step one (to update the address), in theory it won't make sense to go the "next" step, because all three will already be complete. It'll need some nuance here in terms of the architecture, but again I think it'll be best if the client just naively sends it's data and lets the server make the decisions.

Reply:
> but again I think it'll be best if the client just naively sends it's data and lets the server make the decisions.

👍